instrumentation/couchbase/couchbase-2.6/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/couchbase/v2_6/CouchbaseNetworkInstrumentation.java

```diff
@@ -65,9 +64,12 @@ public static void addNetworkTagsToSpan(
       VirtualField<CouchbaseRequest, Span> virtualField =
           VirtualField.find(CouchbaseRequest.class, Span.class);
 
+      // TODO add support for peer service name
```

Comment:
can you open an issue for this, since it's a regression?

Reply:
#5050